local-setup/nginx/sites-enabled/odk.conf

```diff
@@ -34,7 +34,7 @@ server {
   location / {
     proxy_pass              http://odk.aether.local:8002;
 
-    proxy_set_header        Host               $host;
+    proxy_set_header        Host               $host:$server_port;
```

Comment:
We should probably explicitly say `8443`, or change all other occurences of `8443` to `$server_port`, otherwise this might be confusing in 6 months time

Reply:
I used `$server_port` in order to avoid hardcoding the nginx port in this config file. Personally, I find `$server_port` less confusing, but that's just me. What do you think @obdulia-losantos?

Reply:
The problem here is that there are two server ports `80` and `8443` and ODK only accepts the second one...

Reply:
Otherwise I would also prefer the `$server_port` option